src/Microsoft.AspNetCore.Razor.Tools/Client.cs

```diff
@@ -74,6 +79,18 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
+        private static PipeOptions GetPipeOptions()
+        {
+            var options = PipeOptions.Asynchronous;
+
+            if (Enum.IsDefined(typeof(PipeOptions), PipeOptionCurrentUserOnly))
```

Comment:
Does it complain if you add the flag all the time (regardless of the platform)?

Reply:
Yup. Throws ArgumentOutOfRange - invalid flag